rebar.config

```diff
@@ -16,6 +17,11 @@
 %% built (!) though the explicit copying in the `relx` overlay fails.
 %% Addressing such a failure consists of updating the content of
 %% `otp_patches/`.
+%%
+%% The project uses a modified rebar3 which is built using this commit:
+%%
+%% https://github.com/tolbrino/rebar3/commit/7b6a07597eb347294c0588d8435aa647a19a6bce
```

Comment:
This is prone to getting out-of-date. I would favour detailed commit message adding rebar3 e.g. https://github.com/aeternity/aeternity/commit/3493e9ec94faa34bf4d57493c93e168bcf00e765

Reply:
removed